File: tests/fixtures/php/callables_unicode.php
<?php
/* é */ function outer(bool $flag): void {
    if ($flag) {
        $closure = function (bool $inner): void {
            if ($inner) {
            }
        };
    }
    $arrow = fn(bool $value): int => $value ? 1 : 0;
}

class Example {
    public function method(bool $flag): void {
        if ($flag) {
        }
    }
}

File: tests/fixtures/python/callables_unicode.py
@decorator
async def decorated(é):
    if é:
        pass


class Café:
    @decorator
    def method(self):
        return lambda value: value if value else 0


def outer(flag):
    if flag:
        def nested(value):
            if value:
                pass
        callback = lambda value: value if value else 0
